authoring/tools-for-developers.md

```diff
@@ -0,0 +1,80 @@
+# Tools for developers
+
+This section presents some useful tools that can be very helpful for the development workflow.
+
+
+## Code linters
+
+Code linteres are tools that analyze source code and identify programming errors, stylistic errors,
+invalid typing, wrong or missing docstrings, etc.
+
+Some linters that could help you to keep your code in good shape are:
+
+- [flake8](https://flake8.pycqa.org/en/latest/)
+- [mypy](http://mypy-lang.org/)
+- [numpydoc](https://numpydoc.readthedocs.io/en/latest/)
+- [pydocstyle](https://github.com/PyCQA/pydocstyle)
+
+
+## Code stylers
+
+Code stylers are tools that fix styling issues in a file, formatting it automatically. 
```

Comment:
```suggestion
Code stylers are tools that fix styling issues in a file, formatting it automatically.
Code *styling* is different from code *linting* because it doesn't change the functionality
of your code at all, it *only* changes how it looks. 
```